Library/Homebrew/keg_relocate.rb

```diff
@@ -133,9 +133,16 @@ def replace_placeholders_with_locations(files, skip_linkage: false)
 
   def openjdk_dep_name_if_applicable
     deps = runtime_dependencies
+    deps ||= to_formula.deps # the tab is missing, so fall back to the formula
     return if deps.blank?
 
-    dep_names = deps.map { |d| d["full_name"] }
+    # TODO: Remove this when we support fetching bottle manifests from non-ghcr.io domains.
```

Comment:
Can you create a `help wanted` issue for this?

Reply:
#12759